README.md

```diff
@@ -76,10 +76,12 @@ Each image is associated with one or more tags. Below is the list of images and
 - [IntelMPI GPU - cuda10 - Ubuntu 16.04](./base/gpu/intelmpi2018.3-cuda10.0-cudnn7-ubuntu16.04)

```

Comment:
missed here

Reply:
What is missed? The only cuda 10.1 image I have added is openmpi ubuntu 18.04

Reply:
[IntelMPI GPU - **cuda10.0** - Ubuntu 16.04]

Reply:
fixed